2020-10-06 09:25:46,890 INFO    MainThread:14940 [internal.py:wandb_internal():62] W&B internal server running at pid: 14940
2020-10-06 09:25:46,892 DEBUG   HandlerThread:14940 [handler.py:handle_request():54] handle_request: login
2020-10-06 09:25:46,893 DEBUG   SenderThread:14940 [sender.py:send():88] send: request
2020-10-06 09:25:46,893 DEBUG   SenderThread:14940 [sender.py:send_request():97] send_request: login
2020-10-06 09:25:46,896 DEBUG   SenderThread:14940 [connectionpool.py:_new_conn():815] Starting new HTTPS connection (1): api.wandb.ai:443
2020-10-06 09:25:47,113 WARNING MainThread:14940 [internal.py:wandb_internal():116] Internal process interrupt: 1
2020-10-06 09:25:47,501 WARNING MainThread:14940 [internal.py:wandb_internal():116] Internal process interrupt: 2
2020-10-06 09:25:47,501 ERROR   MainThread:14940 [internal.py:wandb_internal():119] Internal process interrupted.
2020-10-06 09:25:47,531 DEBUG   SenderThread:14940 [connectionpool.py:_make_request():396] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2020-10-06 09:25:47,533 INFO    SenderThread:14940 [sender.py:send_request_login():144] Login server info: {'cliVersionInfo': {'min_cli_version': '0.8.20', 'max_cli_version': '0.10.4'}}
2020-10-06 09:25:47,534 INFO    SenderThread:14940 [sender.py:finish():573] shutting down sender
2020-10-06 09:25:47,893 INFO    HandlerThread:14940 [handler.py:finish():267] shutting down handler
2020-10-06 09:25:47,893 INFO    MainThread:14940 [internal.py:handle_exit():137] Internal process exited
